Fix error like

/home/kraj/work/angstrom/sources/openembedded-core/build/tmp-eglibc/work/mips64-oe-linux/eglibc-2.16-r2+svnr19383/build-mips64-oe-linux/libc_pic.os: In function `_IO_new_file_fopen':
/home/kraj/work/angstrom/sources/openembedded-core/build/tmp-eglibc/work/mips64-oe-linux/eglibc-2.16-r2+svnr19383/eglibc-2_16/libc/libio/fileops.c:431: undefined reference to `_IO_file_close_it_internal'
collect2: error: ld returned 1 exit status


Signed-off-by: Khem Raj <raj.khem@gmail.com>
Upstream-Status: Pending
Index: libc/libio/fileops.c
===================================================================
--- libc.orig/libio/fileops.c	2012-07-04 18:25:47.000000000 -0700
+++ libc/libio/fileops.c	2012-07-24 00:21:17.220322557 -0700
@@ -428,7 +428,7 @@
 	  result->_mode = 1;
 #else
           /* Treat this as if we couldn't find the given character set.  */
-          (void) INTUSE(_IO_file_close_it) (fp);
+          (void) _IO_file_close_it (fp);
           __set_errno (EINVAL);
           return NULL;
 #endif
